packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -496,6 +498,17 @@ export default class CodeSubmode extends Component<Signature> {
     return undefined;
   }
 
+  private get displayPlayground() {
```

Comment:
Consider renaming to `shouldDisplayPlayground` to avoid confusion with the verb usage of "display"